walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java

```diff
@@ -48,10 +48,12 @@ public int compare(TimeRange a, TimeRange b) {
 
   private final int start;
   private final int duration;
+  private boolean isValid;
```

Comment:
nit: is there a different term that you could use here than valid? (Eg what does "valid" mean? Could be something like: positive duration?) As a suggestion: "isSchedulable"?